packages/app/src/components/challenges/mutations/CreateChallengeMutation.tsx

```diff
@@ -0,0 +1,77 @@
+import { graphql } from 'react-relay';
+
+import {
+    SelectorStoreUpdater,
+    RecordSourceSelectorProxy,
+    ConnectionHandler
+} from 'relay-runtime';
+
+import { connectionUpdater } from '../../../relay';
+import { ROOT_ID } from 'relay-runtime';
+
+export const CreateChallenge = graphql`
+    mutation CreateChallengeMutation($input: CreateChallengeInput!) {
+        CreateChallenge(input: $input) {
+            error
+            challengeEdge {
+                node {
+                    _id
+                    id
+                    title
+                    content
+                    positiveOptions
+                    negativeOptions
+                    address
+                }
+            }
+        }
+    }
+`;
+
```

Comment:
Let's delete the updater here since we dont use it

Reply:
Should I remove optimisticUpdater too?

Reply:
Yes! Just remove the comments from everywhere involving the updaters